lib/matplotlib/lines.py

```diff
@@ -272,6 +272,7 @@ def __init__(self, xdata, ydata,
                  linewidth=None,  # all Nones default to rc
                  linestyle=None,
                  color=None,
+                 secondcolor=None,
```

Comment:
I tend to go for `second_color`. Without context, I'd certainly go for that because it's more readable. Consistency: We have both variants in our API e.g. `markerfacecolor` here and `Patch.set_hatch_color`.